packit_service/worker/jobs.py

```diff
@@ -83,6 +85,9 @@ def process_jobs(self, event: Event) -> Dict[str, HandlerResults]:
         """
         Run a job handler (if trigger matches) for every job defined in config.
         """
+
+        logger.warning("PROCESS JOBS")
```

Comment:
!!!!!!

Reply:
(this was a joke, to be honest :D I just found it amusing the log message was written in all caps)